syntax = "proto2";

package g2p;

enum TaskType {
    TRAIN = 0;
    EVALUATE = 1;
    NONE = 2;
    SHUTDOWN = 3;
}

